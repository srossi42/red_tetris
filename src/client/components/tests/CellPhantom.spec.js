import React from 'react'
import { shallow, configure } from 'enzyme';
import CellPhantom from '../CellPhantom';
import toJson from "enzyme-to-json"


describe('<Cell/> Component', () => {
  it('rendering correctly Cell with color', () => {
    const TETROMINOS = {
      0: { color: "255, 255, 255" },
      I: { color: "80, 227, 230" },
      J: { color: "36, 95, 223" },
      L: { color: "223, 173, 36" },
      O: { color: "223, 217, 36" },
      S: { color: "48, 211, 56" },
      T: { color: "132, 61, 198" },
      Z: { color: "227, 78, 78" },
      W: { color: "128, 128, 128"}
    };
    let wrapper = shallow(
      <CellPhantom type={'T'} color={TETROMINOS['T'].color}/>
    )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
