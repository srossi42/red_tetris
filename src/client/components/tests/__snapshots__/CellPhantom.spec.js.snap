// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cell/> Component rendering correctly Cell with color 1`] = `
<styled.div
  color="132, 61, 198"
  type="T"
/>
`;
